lib/ret_web/controllers/api/v1/hub_controller.ex

```diff
@@ -1,16 +1,101 @@
 defmodule RetWeb.Api.V1.HubController do
   use RetWeb, :controller
+  import RetWeb.ApiHelpers
 
   alias Ret.{Hub, Scene, SceneListing, Repo}
 
   import Canada, only: [can?: 2]
+  import Ecto.Query, only: [where: 3, preload: 2]
 
   # Limit to 1 TPS
   plug(RetWeb.Plugs.RateLimit)
 
   # Only allow access to remove hubs with secret header
   plug(RetWeb.Plugs.HeaderAuthorization when action in [:delete])
 
+  @show_request_schema %{
+                         "type" => "object",
+                         "properties" => %{
+                           "created_by_account_email" => %{
+                             "type" => "string",
+                             "format" => "email"
+                           },
+                           "hub_sids" => %{
```

Comment:
Let's change this to `room_ids` since I think with this API we are remapping the term `hub` to `room`.

Reply:
Also we need to put a limit on the number of rooms you can query for, otherwise this will be abused.